src/Spectre.libGenetic/InconsistentMinimalAndMaximalFillupException.h

```diff
@@ -0,0 +1,40 @@
+/*
+* InconsistentMinimalAndMaximalFillupException.h
+* Thrown when minimal fillup is greater than maximal.
+*
+Copyright 2017 Grzegorz Mrukwa, Wojciech Wilgierz
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include "Spectre.libException/InconsistentArgumentSizesException.h"
+
+namespace spectre::algorithm::genetic
+{
+/// <summary>
+/// Thrown when minimal and maximal fillup values are inconsistent.
+/// </summary>
+class InconsistentMinimalAndMaximalFillupException :
+    public core::exception::InconsistentArgumentSizesException
```

Comment:
This exception is not intended for such a case. It is for collections that should be of equal size. I'd suggest using `ArgumentOutOfRangeException` rather than this.

Reply:
done

Reply:
ArgumentOutOfRangeException is also not exactly this error. I will create new error for this purpose from BaseException

Reply:
Good idea.